src/sbt-test/lagom-conductr-bundle/lagom-java-service-execution/test

```diff
@@ -0,0 +1,17 @@
+##
+## This test starts a sandbox, deploys a LagomJava service and executes an HTTP request
+## through the Service Gateway to ensure the service is reachable.
+##
+
+> sandbox run 2.0.2
+
+> install
+
+> verifyConductInfo
+
+> verifyIsStarted lagom-service-impl
+
+$ sleep 5000
+> assertRequest 9000 /foo foo-response-hardcoded in FooServiceImpl
```

Comment:
I agree. Also, the sleep should not be necessary. `verifyIsStarted` should only return true if the application has signalled that it is started and able to process requests.